<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>421</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="newbutton">
         <property name="text">
          <string>새 게임</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continuebutton">
         <property name="text">
          <string>이어하기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replaybutton">
         <property name="text">
          <string>리플레이</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="endbutton">
         <property name="text">
          <string>게임 종료</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2024402049 정하영</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>불러오기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="boardContainer" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>700</width>
        <height>700</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
